CodePush.js

```diff
@@ -1,15 +1,13 @@
 'use strict';
 
-var { Alert } = require("./AlertAdapter");
-var NativeCodePush = require("react-native").NativeModules.CodePush;
-var PackageMixins = require("./package-mixins")(NativeCodePush);
-var requestFetchAdapter = require("./request-fetch-adapter.js");
-var Sdk = require("code-push/script/acquisition-sdk").AcquisitionManager;
-var semver = require("semver");
+import { Alert } from "./AlertAdapter";
+let NativeCodePush = require("react-native").NativeModules.CodePush;
```

Comment:
Could you use object destructuring to pull out the CodePush module using import syntax? Something like this: import { NativeModules: { CodePush as NativeCodePush } } from "react-native";